core/controllers/ingest.py

```diff
@@ -80,7 +80,14 @@ def ingest(body, excel_file):
 
     save_submission_file(excel_file, submission_id)
 
-    return "Success: Spreadsheet data ingested", 200
+    return jsonify(
+        {
+            "detail": "Spreadsheet successfully uploaded",
+            "status": 200,
+            "validation_errors": {},
```

Comment:
The current example doesn't include `validation_errors` and I don't think we need to include it as it will always be empty and we determine if the ingest is successful from the 200 status.

Reply:
I've fixed both comments here